src/app/core/services/node.ts

```diff
@@ -97,6 +104,7 @@ export class NodeService {
       data: {
         initialClusterData: cluster,
         initialNodeData: {
+          operatingSystemProfile: md.annotations[OPERATING_SYSTEM_PROFILE_ANNOTATION],
```

Comment:
I think we should add a null check here on `md.annotations` 
```suggestion
          operatingSystemProfile: md.annotations?.[OPERATING_SYSTEM_PROFILE_ANNOTATION],
```